apps/theming-designer/src/components/SemanticSlots.tsx

```diff
@@ -1,13 +1,168 @@
+    'buttonBackgroundDisabled', // neutralLighter
+    'buttonBorder', // 'transparent'
+    'buttonText', // neutralPrimary
+    'buttonTextHovered', // neutralDark
+    'buttonTextChecked', // neutralDark
+    'buttonTextCheckedHovered', // black
+    'buttonTextPressed', // neutralDark
+    'buttonTextDisabled', // neutralTertiary
+    'buttonBorderDisabled', // 'transparent'
+
+    'primaryButtonBackground', // themePrimary
+    'primaryButtonBackgroundHovered', // themeDarkAlt
+    'primaryButtonBackgroundPressed', // themeDark
+    'primaryButtonBackgroundDisabled', // neutralLighter
+    'primaryButtonBorder', // 'transparent'
+    'primaryButtonText', // white
+    'primaryButtonTextHovered', // white
+    'primaryButtonTextPressed', // white
+    'primaryButtonTextDisabled', // neutralQuaternary
+
+    'accentButtonBackground', // accent
+    'accentButtonText', // white
+    // INPUTS - 14
+    'inputBorder', // neutralTertiary
+    'inputBorderHovered', // neutralPrimary
+    'inputBackground', // white
+    'inputBackgroundChecked', // themePrimary
+    'inputBackgroundCheckedHovered', // themeDarkAlt
+    'inputForegroundChecked', // white
+    'inputFocusBorderAlt', // themePrimary
+    'smallInputBorder', // neutralSecondary
+    'inputText', // neutralPrimary
+    'inputTextHovered', // neutralDark
+    'inputPlaceholderText', // neutralSecondary
+    'disabledBackground', // neutralLighter
+    'disabledText', // neutralTertiary
+    'disabledSubText', // neutralQuaternary
+    // LISTS - 7
+    'listBackground', // white
+    'listText', // neutralPrimary
+    'listItemBackgroundHovered', // neutralLighter
+    'listItemBackgroundChecked', // neutralLight
+    'listItemBackgroundCheckedHovered', // neutralQuaternaryAlt
+    'listHeaderBackgroundHovered', // neutralLighter
+    'listHeaderBackgroundPressed', // neutralLight
+    // MENUS - 8
+    'menuBackground', // white
+    'menuDivider', // neutralTertiaryAlt
+    'menuIcon', // themePrimary
+    'menuHeader', // themePrimary
+    'menuItemBackgroundHovered', // neutralLighter
+    'menuItemBackgroundPressed', // neutralLight
+    'menuItemText', // neutralPrimary
+    'menuItemTextHovered' // neutralDark
+  ];
+
+  const semanticSlotWidget = (color: string, name: string): JSX.Element => {
     return (
-      <Card styles={{ root: { minWidth: '800px', maxWidth: '1200px', height: 'auto' } }}>
-        <h1>Semantic Slots</h1>
-      </Card>
+      <div key={name} className={slotClassName}>
+        <Stack horizontal gap={5}>
+          <div key={name} className={semanticPaletteColorBox} style={{ backgroundColor: color }} />
+          <div>{name}</div>
+        </Stack>
+      </div>
     );
-  }
-}
+  };
+
+  // fill noneSlots
+  const fillVariantSlotsList = (variantType: VariantThemeType): JSX.Element[] => {
+    // call getVariant to get the default
+    let noneThemeVariant: ITheme;
+    if (props.theme) {
+      noneThemeVariant = getVariant(props.theme!, variantType);
```

Comment:
Should check if theme is defined or make it not optional. (ie avoid the `!` in `props.theme!`)

Reply:
done.